api/v1alpha1/nstemplatetier_types.go

```diff
@@ -22,6 +22,17 @@ type NSTemplateTierSpec struct {
 	// +optional
 	// +mapType=atomic
 	SpaceRoles map[string]NSTemplateTierSpaceRole `json:"spaceRoles,omitempty"`
+
+	// Provides the name of the Service Account whose token is to be copied
+	// +optional
+	SpaceRequestConfigName SpaceRequestConfig `json:"spaceRequestConfig,omitempty"`
```

Comment:
```suggestion
	SpaceRequestConfigName *SpaceRequestConfig `json:"spaceRequestConfig,omitempty"`
```
for `omitempty` to work it has to be a pointer. I faced a similar issue here https://github.com/codeready-toolchain/api/pull/382

Reply:
Yeah, thanks. Updated.